### ISSUE TITLE
Holes in "The Bridge" barrier

### ISSUE THREAD

**Issue by rubenwardy** [MT-CTF/capturetheflag#84]

**Comment by afshaan4** [comment]
yeah i spotted one at (692, -17, 1)
![hole_in_wall](https://user-images.githubusercontent.com/20749736/35471110-29831562-037b-11e8-8167-8738d24893dc.png)

**Comment by Gaara95** [comment]
When I was making a map I found those were where the grass and other random stuff were, so I had to remove them and then redo the barrier thing.

**Comment by ClobberXD** [comment]
That's most likely the culprit - there must've been grass along the barrier in the original map, where the holes are - and since CTF has disabled unnecessary nodes and items (e.g. flint, grass) for extra focus, the grass is removed in-game, and voila - the ugly holes in the barrier.

A solution to this would be to **re-emerge the barriers** in the original schem file (located in `/mods/ctf_map/02_bridge.mts`)

**Comment by afshaan4** [comment]
No more convenient teleporting back to spawn after collecting weapons.

**Comment by ClobberXD** [comment]
> No more convenient teleporting back to spawn after collecting weapons.

Sadly, that issue won't be addressed until the barriers have been modified (I've made a relevant commit, but @rubenwardy prefers a more elegant solution, so it's on hold for now)

**Comment by afshaan4** [comment]
what if when the map is being modified for ctf the tool first removes all these extra things *then* places the barrier.

**Comment by ClobberXD** [comment]
Good idea - a list of replaceable nodes can be created, and if a node matches an entry in the list, the barrier will remove it and emerge right through...

**Comment by Gaara95** [comment]
This didn't happen with the trees tho strange

**Comment by rubenwardy** [comment]
Trees exist in this subgame, flowers don't

**Comment by ClobberXD** [comment]
> This didn't happen with the trees tho strange

The original CTF map is full of trees BTW... :P

Only non-tree plant-like nodes / items like mushrooms, flowers, grass, and the like have been disabled.

**Comment by ClobberXD** [comment]
Reopening, as this is the correct repo.
